lib/public/components/common/selection/infoLoggerButtonGroup/infoLoggerButtonGroup.js

```diff
@@ -11,42 +11,47 @@
  *  or submit itself to any jurisdiction.
  */
 
-import { frontLink } from '../../navigation/frontLink.js';
-import { dropdown } from '../../popover/dropdown.js';
-import { iconCaretBottom } from '/js/src/icons.js';
 import { h } from '/js/src/index.js';
 import { getInfologgerLink } from '../../../../services/externalRouting/getInfologgerLink.js';
 import { CopyToClipboardComponent } from './CopyToClipboardComponent.js';
+import { identifierColumnAndActionsDropdown } from './identifierColumnAndActionsDropdown.js';
+
+/**
+ * Generates an infologger link component with the given infologger link and label.
+ *
+ * @param {string} infologger The infologger link.
+ * @param {string} label The label for the link.
+ * @return {Component|null} The infologger link component, or null if the infologger is not available.
+ */
+const generateInfologgerLink = (infologger, label) => infologger && h('a', { href: infologger, target: '_blank' }, label);
```

Comment:
I think generate is a bit too much, `getInfologgerLink` may be enough? Or even `infologgerLink`

The parameter `infologger` seems a bit misleading, I would put `infologgerUrl`